python/lsst/analysis/tools/actions/plot/focalPlanePlot.py

```diff
@@ -54,7 +54,10 @@ class FocalPlanePlot(PlotAction):
     zAxisLabel = Field[str](doc="Label to use for the z axis.", optional=False)
 
     nBins = Field[int](
-        doc="Number of bins to use within the effective plot ranges along the spatial directions.",
+        doc="Number of bins to use within the effective plot ranges along"
+        " the spatial directions. If nBins is negative the number of"
+        " bins is adapted to the source density, with lower densities"
+        " using fewer bins, but with a minimum of abs(nBins).",
```

Comment:
I think that instead of using negative it would be better to have a config option, something like doUseAdaptiveBinning, that is more explicit and easier to understand at a glance. Then the normal bins config can stay the same and be used for both.